Misc/NEWS.d/next/Build/2022-10-28-22-24-26.gh-issue-98831.IXRCRX.rst

```diff
@@ -1 +1,10 @@
-We have new tooling, in ``Tools/cases_generator``, to generate the interpreter switch from a list of opcode definitions.
+We have new tooling, in ``Tools/cases_generator``,
+to generate the interpreter switch from a list of opcode definitions.
+This affects only people who are adding or modifying instruction definitions.
+The instruction definitions now live in ``Python/bytecodes.c``,
+in the form of a custom DSL (under development).
```

Comment:
```suggestion
in the form of a `custom DSL (under development)
<https://github.com/faster-cpython/ideas/blob/main/3.12/interpreter_definition.md>`__.
```

I found it a little hard to follow that the DSL was introduced but not described here, then there was a not directly related sentence, and then a bare link to the DSL. I suggest just linking it here, which is also more concise.